planner/core/logical_plans.go

```diff
@@ -1219,6 +1222,7 @@ type LogicalCTE struct {
 type LogicalCTETable struct {
 	logicalSchemaProducer
 
+	seedPlan     LogicalPlan
```

Comment:
Why do we need to add it here? and only add the seed part without the recursive part together